tests/sentry/utils/test_pickle_protocol.py

```diff
@@ -0,0 +1,27 @@
+import pickle
+from pickle import PickleBuffer, PickleError
+
+from sentry.testutils import TestCase
+
+
+class PickleProtocolTestCase(TestCase):
+    """
+    At the time of adding this test we still monkey patch `pickle` and hardcode the protocol to be 2.
+    For legacy reasons see `src/sentry/monkey/pickle.py`.
+
+    This test is for a change that's being made to allow explicitly passing a newer protocol to
+    pickle. If we remove the monkey patching to pickle there is no longer a need for this test.
+
+    """
+
+    def test_pickle_protocol(self):
```

Comment:
cc @markstory @wedamija Added this test since `PickleBuffer` requires protocol 5 or above.